rllib/BUILD

```diff
@@ -1861,6 +1861,27 @@ py_test(
     srcs = ["core/models/tests/test_catalog.py"]
 )
 
+py_test(
```

Comment:
Unified tests between tf and torch to be able to compare exact number of model parameters (assert architectures are the same).